kong/plugins/ai-request-transformer/handler.lua

```diff
@@ -0,0 +1,72 @@
+local _M = {}
+
+-- imports
+local kong_meta     = require "kong.meta"
+local fmt           = string.format
+local llm           = require("kong.llm")
+--
+
+_M.PRIORITY = 777
+_M.VERSION = kong_meta.version
+
+local function bad_request(msg)
+  kong.log.warn(msg)
```

Comment:
See: https://github.com/Kong/kong/pull/12337#discussion_r1462442564

```suggestion
  kong.log.info(msg)
```